DigitalLearningSolutions.Web.Tests/Services/UserServiceTests.cs

```diff
@@ -187,22 +198,42 @@ public void
                         accountDetailsData.ProfessionalRegistrationNumber,
                         accountDetailsData.HasBeenPromptedForPrn,
                         accountDetailsData.JobGroupId,
-                        detailsLastChecked,
+                        currentTime,
+                        null,
                         accountDetailsData.UserId,
+                        isPrimaryEmailUpdated,
                         shouldUpdateProfileImage
                     )
                 )
                 .MustHaveHappened();
-            A.CallTo(
-                    () => userDataService.SetCentreEmail(
-                        accountDetailsData.UserId,
-                        centreId,
-                        centreEmail,
-                        null,
-                        A<IDbTransaction?>._
+
+            if (isCentreEmailUpdated)
+            {
+                A.CallTo(
+                        () => userDataService.SetCentreEmail(
+                            accountDetailsData.UserId,
+                            centreId,
+                            centreEmail,
+                            null,

```

Comment:
this is only null because `shouldUpdateProfileImage` is true. if it were false, it would be `currentTime`